lib/validators/AlgorandTxValidator.js

```diff
@@ -1,47 +1,101 @@
+const BigNumber = require('bignumber.js')
+
 class AlgorandTxValidator {
- 
   constructor(params){
     this.envs = params.envs,
     this.blockchain = params.blockchain
+    this.skipAppIndexTypes = ['FundOptIn']
+    this.valid = false
+    this.markAs = null
+    this.error = null
   }
 
   async isTransactionValid(blockchainTransaction, hotWalletAddress) {
     if (typeof blockchainTransaction.txRaw !== 'string') { return { valid: false } }
-
+    const { txRaw, blockchainTransactableType, blockchainTransactables } = blockchainTransaction
     try {
       const transaction = JSON.parse(blockchainTransaction.txRaw)
+      const transactableType = blockchainTransactableType || blockchainTransactables[0].blockchainTransactableType
```

Comment:
I think `blockchainTransactableType` would always be null here, since all of the relations currently build via `blockchainTransactables`

We probably need to update API on Comakery-server as well, to reflect the change.